build.sbt

```diff
@@ -82,20 +84,42 @@ lazy val `quill-finagle-mysql` =
     )
     .dependsOn(`quill-sql-jvm` % "compile->compile;test->test")
 
-lazy val `quill-async` = 
+lazy val `quill-async` =
   (project in file("quill-async"))
     .settings(commonSettings: _*)
     .settings(mimaSettings: _*)
     .settings(
       fork in Test := true,
       libraryDependencies ++= Seq(
-        "com.github.mauricio" %% "db-async-common"  % "0.2.20",
-        "com.github.mauricio" %% "mysql-async"      % "0.2.20",
-        "com.github.mauricio" %% "postgresql-async" % "0.2.20"
+        "com.github.mauricio" %% "db-async-common"  % "0.2.20"
       )
     )
     .dependsOn(`quill-sql-jvm` % "compile->compile;test->test")
 
+lazy val `quill-async-mysql` =
+  (project in file("quill-async-mysql"))
+    .settings(commonSettings: _*)
+    .settings(mimaSettings: _*)
+    .settings(
+      fork in Test := true,
+      libraryDependencies ++= Seq(
+        "com.github.mauricio" %% "mysql-async"      % "0.2.20"
+      )
+    )
+    .dependsOn(`quill-async` % "compile->compile;test->test")
+
+lazy val `quill-async-postgresql` =
```

Comment:
Could you rename this project to `quill-async-postgres`? We normally use `postgres` (e.g. `PostgresAsyncContext`, `PostgresDialect`)